db/seeds.d/50_discovery_templates.rb

```diff
@@ -1,8 +1,8 @@
-kind = TemplateKind.find_or_create_by_name('kexec')
+kind = TemplateKind.find_or_create_by(:name =>'kexec')
```

Comment:
Do you care about Rails 3 compatibility in this PR? If so, substitute this by `where(conditions).first_or_create` like we did in Foreman core https://github.com/theforeman/foreman/pull/2747/files